doc/en/components/grids/_shared/conditional-cell-styling.md

```diff
@@ -0,0 +1,684 @@
+        <span *ngIf="cell.row.data.UnitPrice != 0">${{val}}</span>
+    </ng-template>
+</igx-column>
+```
+<!-- end: Angular -->
+
+```razor
+Add tree grid example
+```
+
+<!-- ComponentEnd: TreeGrid -->
+
+<!-- ComponentStart: HierarchicalGrid -->
+```razor
+add example
+```
+
+<!-- ComponentEnd: HierarchicalGrid -->
+
+The `CellClasses` input accepts an object literal, containing key-value pairs, where the key is the name of the CSS class, while the value is either a callback function that returns a boolean, or boolean value.
+
+<!-- ComponentStart: Grid -->
+
+```typescript
+
+// sample.component.ts
+
+private upFontCondition = (rowData: any, columnKey: any): boolean => {
+    return rowData[columnKey] > 95;
+}
+
+private downFontCondition = (rowData: any, columnKey: any): boolean => {
+    return rowData[columnKey] <= 95;
+}
+
+public beatsPerMinuteClasses = {
+    downFont: this.downFontCondition,
+    upFont: this.upFontCondition
+};
+```
+
+```razor
+add example
+```
+
+<!-- Angular -->
+```scss
+// sample.component.scss
+
+::ng-deep {
+    .upFont {
+        color: green;
+    }
+
+    .downFont {
+        color: red;
+    }
+}
+```
+<!-- end: Angular -->
+
+<!-- ComponentEnd: Grid -->
+
+<!-- ComponentStart: TreeGrid -->
+
+```typescript
+// sample.component.ts
+
+private upPriceCondition = (rowData: any, columnKey: any): boolean => {
+    return rowData[columnKey] > 25;
+}
+
+private downPriceCondition = (rowData: any, columnKey: any): boolean => {
+    return rowData[columnKey] <= 25;
+}
+
+public priceClasses = {
+    downPrice: this.downPriceCondition,
+    upPrice: this.upPriceCondition
+};
+```
+
+```razor
+Add treegrid example
+```
+
+```scss
+// sample.component.scss
+
+::ng-deep {
+    .upPrice {
+        color: red;
+    }
+
+    .downPrice {
+        color: green;
+    }
+}
+```
+
+<!-- ComponentEnd: TreeGrid -->
+
+<!-- ComponentStart: HierarchicalGrid -->
+
+```razor
+Add hierarchical grid example
+```
+<!-- ComponentEnd: HierarchicalGrid -->
+
+<!-- Angular -->
+Use **::ng-deep** or **ViewEncapsulation.None** to force the custom styles down through the current component and its children.
+<!-- end: Angular -->
+
+### Demo
+
+<!-- ComponentStart: Grid -->
+
+<code-view style="height:620px"
+           data-demos-base-url="{environment:demosBaseUrl}"
+           iframe-src="{environment:demosBaseUrl}/{ComponentSample}-cell-styling"
+           github-src="{ComponentSample}/cell-styling" >
+</code-view>
+
+<!-- ComponentEnd: Grid -->
+
+
+<!-- ComponentStart: TreeGrid -->
+
+<code-view style="height:600px"
+           data-demos-base-url="{environment:demosBaseUrl}"
+           iframe-src="{environment:demosBaseUrl}/{ComponentSample}-conditional-cell-style"
+           github-src="{ComponentSample}/conditional-cell-style" >
+</code-view>
+
+<!-- ComponentEnd: TreeGrid -->
+
+<!-- ComponentStart: HierarchicalGrid -->
+
+<!-- TODO -->
+
+<!-- ComponentEnd: HierarchicalGrid -->
```

Comment:
replace above 3 code-view blocks with just one code-view and remove component build flag because routing path to the sample will be the same for all grid samples in Sample Browser.